Makefile

```diff
@@ -94,7 +94,12 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
+	go tool vet --shadow ./test
```

Comment:
Please remove this line as well.  Even though it is working in the test folder now.  I don't want future commits struggling with this tool until the issue is sorted.